server/types/express.d.ts

```diff
@@ -5,17 +5,27 @@ declare namespace Express {
       token: string
       refreshToken: string
       refreshTime: any
-      firstName: string
-      lastName: string
-      userId: string
-      displayName: string
-      isReviewer: boolean
-      isCoordinator: boolean
-      activeCaseLoadId: string
```

Comment:
all this was on `res.locals.user`. There is a cut down version of the user on `req.user`